components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -100,10 +129,29 @@ class TextFieldTests: XCTestCase {
 
     XCTAssertTrue(textField.subviews.contains(leftView))
     XCTAssertTrue(textField.subviews.contains(rightView))
+
+    XCTAssertEqual(textField.leadingView, leftView)
```

Comment:
I put in a check. It requires iOS 9 but seeing as we don't have an RTL device testing, much less one that's on iOS 8, I think it'll be fine.